app/components/careers_video_card_component/careers_video_card_component.scss

```diff
@@ -0,0 +1,21 @@
+.careers-video-card {
+  &--content {
```

Comment:
Can we change these to __content, the double hyphens are used for modifiers but these feel more like elements.